datumaro/components/extractor.py

```diff
@@ -127,7 +131,10 @@ def __iter__(self):
 @attrs
 class Label(Annotation):
     _type = AnnotationType.label
-    label = attrib(converter=int)
+    label = attrib(converter=int, default=0)
+    name = attrib(type=str, default=None)
```

Comment:
Name is better to be stored in attributes

Reply:
Resolved